r/R/schema.R

```diff
@@ -85,7 +85,6 @@ Schema <- R6Class("Schema",
   active = list(
     names = function() {
       out <- Schema__field_names(self)
-      # Hack: Rcpp should set the encoding
       Encoding(out) <- "UTF-8"
       out
     },
```

Comment:
If `cpp11` is doing what it claims wrt unicode, you can remove this and just 

```r
names = function() Schema__field_names(self),
```

Reply:
is there a test ?